xmake/core/project/target.lua

```diff
@@ -979,7 +980,9 @@ function _instance:pkgenvs()
                 end
             end
         end
-        for _, pkg in pkgs:keys() do
```

Comment:
we should add `hashset:orderkeys` like table.orderpairs()

Reply:
I'm not sure how to do that in a better way than making a table and sorting it